website/pages/blog/announcing-the-vercel-source-plugin.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Announcing The Vercel Source Plugin
+tag: announcement
+date: 2022/12/14
+description: >-
+CloudQuery is an open-source, extensible framework that gives you a
+single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+happy to announce the release of the Vercel source plugin.
```

Comment:
```suggestion
  CloudQuery is an open-source, extensible framework that gives you a
  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
  happy to announce the release of the Vercel source plugin.
```

Don't we need to indent this? I remember an issue with a previous PR

Reply:
Correct